File: Typescript/src/findClosest.ts
export function findClosest(x: number, y: number, z: number): number {
	let a = Math.abs(x - z) - Math.abs(y - z);
	if (a < 0) {
		return 1;
	} else if (a > 0) {
		return 2;
	} else {
		return 0;
	}
};

File: Dart/lib/findClosest.dart
class Solution {
  int findClosest(int x, int y, int z) {
    int d = (x - z).abs() - (y - z).abs();
    return (d < 0)
        ? 1
        : (d > 0)
            ? 2
            : 0;
  }
}
